tools/run-bidi-server.mjs

```diff
@@ -58,7 +58,7 @@ if (CHANNEL === 'local') {
 
 // DEBUG = (empty string) is allowed.
 const DEBUG = process.env.DEBUG ?? 'bidi:*';
-const DEBUG_COLORS = process.env.DEBUG_COLORS || 'yes';
+const DEBUG_COLORS = process.env.DEBUG_COLORS || 'false';
```

Comment:
Setting it to `false` by default as it makes the log generated files unreadable.

Reply:
![image](https://github.com/GoogleChromeLabs/chromium-bidi/assets/34244704/ef411916-f045-4601-aab1-64a9b27d7a49)